napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -803,14 +1048,18 @@ def _add_to_installed(distname, enabled, npe_version=1):
                 meta = {}
 
             self.installed_list.addItem(
-                PackageMetadata(
-                    metadata_version="1.0",
-                    name=norm_name,
-                    version=meta.get('version', ''),
-                    summary=meta.get('summary', ''),
-                    home_page=meta.get('url', ''),
-                    author=meta.get('author', ''),
-                    license=meta.get('license', ''),
+                (
+                    PackageMetadata(
+                        metadata_version="1.0",
+                        name=norm_name,
+                        version=meta.get('version', ''),
+                        summary=meta.get('summary', ''),
+                        home_page=meta.get('Home-page', ''),
```

Comment:
Just curious: how come we've changed to retrieving `Home-page` rather than `url` here?

Reply:
This was the keyword in the installed plugins information.  When you asked before about having the website available for the installed plugins list, I realized it should have been showing one.  The reason it wasn't was because in the meta data, it was under the keyword `Home-page`.  So, this is already happening somewhere upstream.